spec/models/hmcts_common_platform/defendant_summary_spec.rb

```diff
@@ -79,7 +79,7 @@
 
     let(:defendant_summary) do
```

Comment:
it looks like this spec test need a context:
```
context "when masterDefendantId matches defendantId"
  ...
  it "includes only summaries"
  ...
  end
end
```

Reply:
Ah, but in this context masterDefendantId doesn't match defendantId. defendantId isn't read at all.